torch_xla/csrc/ops/dynamic_ir.cpp

```diff
@@ -35,13 +35,18 @@ SizeNode::SizeNode(torch::lazy::Value input, size_t dim)
 };
 
 int64_t SizeNode::getDynamicValue() const {
+  if (dyanmic_value_computed_) {
+    XLA_COUNTER("CachedSizeNodeValue", 1);
+    return runtime_size_;
```

Comment:
Will this runtime_size_ ever be updated once set?

Reply:
it should be impossible. We don't allow in place update of the node.